src/util/states.ts

```diff
@@ -450,6 +452,7 @@ export function blurSeries(
     }
 
     const targetSeriesModel = ecModel.getSeriesByIndex(targetSeriesIndex);
+    const targetStack = targetSeriesModel.option.type === 'bar' && (targetSeriesModel.option as BarSeriesOption).stack;
```

Comment:
Please use `targetSeriesModel.get('type')` instead of accessing `option` directly.